documentation/cubic-pos-article.md

```diff
@@ -0,0 +1,155 @@
+# Introduction
+
+Namada is a L1 proof-of-stake blockchain that provides interchain asset-agnostic privacy. Namada is based on a byzantine fault-tolerant (BFT) consensus mechanism widely used in the Cosmos ecosystem, provided by Tendermint ([Go implementation]((https://github.com/tendermint/tendermint)) with the [rust bindings]((https://github.com/informalsystems/tendermint-rs)))., that enables fast block finality times and much lower computational overhead than in proof-of-work (PoW) consensus mechanisms, as in Bitcoin. For consensus to be achieved in a BFT system, at least a 2/3 majority of the nodes (called validators) must be honest and behave according to the protocol.
+
+Namada uses proof-of-stake (PoS) to provide security against Sybil attacks on the network, wherein a malicious entity achieves enough voting power to disrupt consensus of the true chain. Rather than miners competing to create the next block by hashing with lots of computing power in PoW systems, the block proposer is selected from a set of validators with probability proportional to the validator's bonded stake (or voting power). This voting power is then used by the validators to determine if the 2/3 majority of consensus has been achieved. Furthermore, the block proposer mechanism also allows PoS chains to be much greener than PoW chains that need to consume massive amounts of energy to reach consensus.
```

Comment:
Removing "true." As for "much greener," what kind of stat do you recommend? As before, I don't think providing actual numbers matter much here (unless we could say precisely some percentage decrease in energy grid usage, which I'm not sure we can), and after all this is just the introduction rather than the focus of the article.